src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -618,6 +622,15 @@ private void processLogsAndUpload() throws InterruptedException {
                                     startPosition = processingFileInformation.startPosition;
                                 }
                             }
+
+                            //TODO: setting this flag is only to develop incrementally without having to changed all
+                            // tests yet, so that we can avoid a massive PR. This will be removed in the end.
+                            if (ACTIVE_LOG_FILE_FEATURE_ENABLED_FLAG.get()) {
```

Comment:
This boolean value is only for giving a chance for this small PR to perform some tests without changing the current logic, such that all tests can pass and this small PR can be merged. This is only for serving the purpose of avoiding massive PR, and this static boolean constant will be removed in the end.